test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -415,8 +414,7 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 			}
 		})
 		It("[test_id:55012] Should have the correct RPS configuration", func() {
-			if profile.Spec.WorkloadHints != nil && profile.Spec.WorkloadHints.RealTime != nil &&
-				!*profile.Spec.WorkloadHints.RealTime && !profileutil.IsRpsEnabled(profile) {
+			if !profileutil.IsRpsEnabled(profile) {
```

Comment:
@MarSik don't we want to check the conditions of whether workloadhints is enabled or if RealTime workloadhints is enabled ?